app/models/scan_item.rb

```diff
@@ -54,15 +54,15 @@ def self.seed
 
   def self.preload_default_profile
     # Create sample VM scan profiles
-    vm_profile = ScanItemSet.find_or_initialize_by_name(SAMPLE_VM_PROFILE)
+    vm_profile = ScanItemSet.find_or_initialize_by(:name => SAMPLE_VM_PROFILE[:name])
```

Comment:
simpler is `vm_profile = ScanItemSet.find_or_initialize_by(SAMPLE_VM_PROFILE.slice(:name))`